crates/core/c8y_api/src/smartrest/smartrest_serializer.rs

```diff
@@ -82,6 +81,7 @@ pub fn succeed_operation(
 
 #[derive(Debug, Copy, Clone)]
 pub enum CumulocitySupportedOperations {
+    C8ySoftwareList,
```

Comment:
`c8y_SoftwareList` is a very old operation replaced by `c8y_SoftwareUpdate` long time ago. If we need this, something is wrong (likely confusion of the inventory fragment name `c8y_SoftwareList`).

Reply:
Ah, I didn't know that, it seemed weird that it was missing, now i know the reason :) I'll replace it with a proper usage of `c8y_SoftwareUpdate`